File: 21C037/day7.py

t=int(input())
s=""
for j in range(t):
    n=int(input())
    l=list(input())
    index=0
    s=""+l[0].lower()
    for i in range(len(l)-1):
        if(l[i].lower()==l[i+1].lower()):
            continue
        else:
            s+=l[i+1].lower()

    if(s=="meow"):
        print("YES")
    else:
        print("NO")
        
    

File: 21C089/day7.java
import java.util.Scanner;

public class day7 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        input.nextLine();
        for (int j = 0; j < t; j++) {
            int n = input.nextInt();
            input.nextLine();
            String l = input.nextLine().toLowerCase();
            StringBuilder s = new StringBuilder();
            s.append(l.charAt(0));

            for (int i = 0; i < n - 1; i++) {
                if (Character.toLowerCase(l.charAt(i)) != Character.toLowerCase(l.charAt(i + 1))) {
                    s.append(Character.toLowerCase(l.charAt(i + 1)));
                }
            }

            if (s.toString().equals("meow")) {
                System.out.println("YES");
            } else {
                System.out.println("NO");
            }
        }
    }
}
